plugins/inputs/execd/execd.go

```diff
@@ -75,6 +76,11 @@ func (e *Execd) Start(acc telegraf.Accumulator) error {
 	e.process.ReadStderrFn = e.cmdReadErr
 
 	if err = e.process.Start(); err != nil {
+		// if there was only one argument, and it contained spaces, warn the user
+		// that they may have configured it wrong.
+		if len(e.Command) == 1 && strings.Contains(e.Command[0], " ") {
+			e.Log.Warn("Command contained spaces but no arguments. Check docs to make sure you've configured inputs.execd Command correctly.")
```

Comment:
This warning comes across a little like "RTFM".  I think we can tell the user how to fix the problem instead.

How about this: "The execd input's command setting contained spaces but no arguments.  This setting expects the program and arguments as an array of strings, not as a space delimited string."

Reply:
I like your wording better, but it could possibly be correct, eg: `Command = ["c:\Program Files\Program.exe"] has no arguments, and a space, but is correct.